Cannot build project due to ISPN protoschema and 26.2 branch

### Before reporting an issue

- [x] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

infinispan

### Describe the bug

It's not possible to build the `/model/infinispan` due to the compatibility check for the branch 26.2

```
[INFO] --- proto-schema-compatibility:6.0.7:proto-schema-compatibility-check (default) @ keycloak-model-infinispan ---
[INFO] Backwards compatibility check against local file '/home/mabartos/Documents/RH/keycloak/model/infinispan/proto.lock' passed.
[INFO] Checking backwards compatibility check against remote file 'https://raw.githubusercontent.com/keycloak/keycloak/refs/heads/archive/release/26.0/model/infinispan/proto.lock'
[INFO] Backwards compatibility check against remote file 'https://raw.githubusercontent.com/keycloak/keycloak/refs/heads/archive/release/26.0/model/infinispan/proto.lock' passed
[INFO] Checking backwards compatibility check against remote file 'https://raw.githubusercontent.com/keycloak/keycloak/refs/heads/archive/release/26.1/model/infinispan/proto.lock'
[INFO] Backwards compatibility check against remote file 'https://raw.githubusercontent.com/keycloak/keycloak/refs/heads/archive/release/26.1/model/infinispan/proto.lock' passed
[INFO] Checking backwards compatibility check against remote file 'https://raw.githubusercontent.com/keycloak/keycloak/refs/heads/release/26.2/model/infinispan/proto.lock'
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Keycloak Model Parent 999.0.0-SNAPSHOT:
[INFO] 
[INFO] Keycloak Model Parent .............................. SUCCESS [  0.149 s]
[INFO] Keycloak Database Support .......................... SUCCESS [  2.508 s]
[INFO] Keycloak Database Support - Private classes ........ SUCCESS [  3.743 s]
[INFO] Keycloak Database Support - REST services .......... SUCCESS [  1.337 s]
[INFO] Keycloak Model JPA ................................. SUCCESS [  7.220 s]
[INFO] Keycloak Model Infinispan .......................... FAILURE [  9.419 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  24.504 s
[INFO] Finished at: 2026-06-05T09:46:03+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.infinispan.protostream:proto-schema-compatibility-maven-plugin:6.0.7:proto-schema-compatibility-check (default) on project keycloak-model-infinispan: An error occurred while running protolock: https://raw.githubusercontent.com/keycloak/keycloak/refs/heads/release/26.2/model/infinispan/proto.lock -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :keycloak-model-infinispan

```

It's about the 26.2 release branch rename.

### Version

main

### Regression

- [ ] The issue is a regression

### Expected behavior

Build pass

### Actual behavior

Build fails

### How to Reproduce?

Build the model/infinispan module

### Anything else?

_No response_

--- a/model/infinispan/pom.xml
+++ b/model/infinispan/pom.xml
@@ -122,7 +122,7 @@
                     </execution>
                 </executions>
                 <configuration combine.self="append">
-                    <remoteLockFiles>https://raw.githubusercontent.com/keycloak/keycloak/refs/heads/archive/release/26.0/model/infinispan/proto.lock,https://raw.githubusercontent.com/keycloak/keycloak/refs/heads/archive/release/26.1/model/infinispan/proto.lock,https://raw.githubusercontent.com/keycloak/keycloak/refs/heads/release/26.2/model/infinispan/proto.lock,https://raw.githubusercontent.com/keycloak/keycloak/refs/heads/archive/release/26.3/model/infinispan/proto.lock,https://raw.githubusercontent.com/keycloak/keycloak/refs/heads/release/26.4/model/infinispan/proto.lock</remoteLockFiles>
+                    <remoteLockFiles>https://raw.githubusercontent.com/keycloak/keycloak/refs/heads/archive/release/26.0/model/infinispan/proto.lock,https://raw.githubusercontent.com/keycloak/keycloak/refs/heads/archive/release/26.1/model/infinispan/proto.lock,https://raw.githubusercontent.com/keycloak/keycloak/refs/heads/archive/release/26.2/model/infinispan/proto.lock,https://raw.githubusercontent.com/keycloak/keycloak/refs/heads/archive/release/26.3/model/infinispan/proto.lock,https://raw.githubusercontent.com/keycloak/keycloak/refs/heads/release/26.4/model/infinispan/proto.lock</remoteLockFiles>
                 </configuration>
             </plugin>
             <plugin>

